### DISCUSSION TITLE
Редактирование карт

### DISCUSSION THREAD

**Discussion by Kukushechka7-ezx** [Sokol-Games-GmbH/PixelCombats.PublicApi#80] [Ideas]
Добавить возможность редактировать описание и название карт.

**Comment by forwork10** [comment]
тут специально сделано, чтобы нельзя было изменить карту.
Игровые режимы, списки карт - все это зависит от конкретных карт...
Тоесть определенному ID карты должен однозначно соответствовать проверенная разработчиком режима, годная для режимв карта, чтобы не попортить игровой процесс, если вдруг владелец там что-то поменял.

У нас есть идея чтобы при выкладывании карты хранилась история версий карты с дополнительными ID.
В итоге когда вносите изменения, то старая версия никуда не девается, а просто появляется новая.
В списке карт хранить уже не только ID карты. но и ID ее версии.
Также в списках чтобы отмечалось, что такаято карта устарела на столькото версий и тамже была кнопка - поменять ссылку на новую версию.
Также требует разработки отката версии карты в списке.
Примерно такой фронт работ, чтобы и обновления на карту выкладывать можно было и в режим карты поставлялись только проверенные разработчиками режимов.

**Comment by kkohno** [comment]
все верно выше описано, однако с описанием тут пожалуй можно и менять... просто новое описание не сломает игру, в отличии от изменений геометрии например

**Comment by Kukushechka7-ezx** [comment]
Сделайте смену имени тоже...

**Comment by forwork10** [comment]
нет........
